combat-trainer.lic

```diff
@@ -1979,6 +1982,11 @@ class GameState
     weapon_skill == 'Brawling'
   end
 
+  def brawl_tactics?
+    return true if !brawling?
+    ((weapon_skill == 'Brawling') && (weapon_name == ''))
```

Comment:
Sorry haven't checked the comments here. 
I often brawl while holding a SE for the small boost in parry ability from the SE's balance. This just checks to make sure the tactics combo section doesn't end up training your held weapon instead of brawling.